### ISSUE TITLE
Fedora Logo Request

### ISSUE THREAD

**Issue by solidc0re** [edent/SuperTinyIcons#757]
## The icon I want is:

* Name: Fedora
* URL: https://fedoraproject.org/
* Link to icon: https://fedoraproject.org/wiki/File:Fedora_infinity.png

* Vector version of the logo: https://fedoraproject.org/wiki/Logo/UsageGuidelines#Obtaining_the_Fedora_logo_for_use_by_graphic_designers
* Official style guide or brand guide: https://fedoraproject.org/wiki/Logo/UsageGuidelines

**Comment by kuubeu** [comment]
The guidelines mention that [the logos should not be replicated](https://fedoraproject.org/wiki/Logo/UsageGuidelines#Don't_attempt_to_replicate_the_logos). Should I add the icon regardless?

**Comment by nishitiwari22** [comment]
Hello, I am looking forward to contributing to this. I have a small question regarding this You want to have an icon like Fedora but not Fedora. I mean I can take inspiration from the logo but the logo design should be different. Am I right?

**Comment by Eiim** [comment]
No, I don't think we want logos that aren't official logos. It's not clear whether or not we should add this logo (as that would involve replicating it).

**Comment by Eiim** [comment]
That said, we do have #698 - that was a bit of an exceptional circumstance as there was no way the full logo could fit in 1kb, but this is also a bit of an exceptional circumstance. But there is some precedent for "not-real" logos. Maybe using the [old logo](https://commons.wikimedia.org/wiki/File:Fedora_icon_(2021).svg) would be less problematic?

**Comment by NitinSingh07** [comment]
Sir, Can you please assign this work to me?

**Comment by Munazir151** [comment]
can I work on this?

**Comment by Potherca** [comment]
Regarding the brand guidelines, I asked for clarification over on the the [Design team’s GitLab](https://gitlab.com/fedora/design/team).

Follow https://forge.fedoraproject.org/design/tickets/issues/29 for a resolution (if any).
